pkg/apis/certmanager/v1alpha2/types.go

```diff
@@ -55,6 +55,7 @@ const (
 // Annotation names for CertificateRequests
 const (
 	CRPrivateKeyAnnotationKey = "cert-manager.io/private-key-secret-name"
+	CRRevisionAnnotationKey   = "cert-manager.io/revision"
```

Comment:
Can you add a comment? (and in v1alpha3 & internal)

Reply:
`pkg/controller/expcertificates/util.go` already defines this annotation: `CertificateRevisionAnnotationKey = "cert-manager.io/certificate-revision"`.

1) can you change this to be `certificate-revision` instead (as per design doc)
2) can you remove duplicate definitions? 😄